### PR TITLE
Docs: Flesh out ga4gh wes docs

### PR DESCRIPTION
**Pull request by JeffreyThiessen**
## What does this PR do and why?

This PR adds more information to the GA4GH WES docs

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally

1. Start docs with the following
```bash
cd docs-site
pnpm build
npm run serve
```
2. In a browser, view the new docs pages
3. Check that the information makes sense from an IRIDA Next admin perspective.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: docs-site/docs/configuration/ga4gh_wes.md [hunk 1]
```diff
@@ -4,34 +4,74 @@ id: ga4gh_wes
...
+#### headers: *Optional
 
-This allows for additional authentication or server settings for your specific deployment of GA4GH WES
+Can be used to set new headers and override existing REST API headers.
 
-By default `{ 'Content-Type': 'application/json' }` is set.
+This allows for additional authentication or server settings for your specific deployment of GA4GH WES.
 
-#### server_url_endpoint
+By default `{ 'Content-Type': 'application/json' }` is set per the GA4GH WES standard.
```
Comment (ericenns) [inline docs-site/docs/configuration/ga4gh_wes.md:48]: **nitpick (non-blocking):** There are actually no default headers, I removed that header when we updated to use multipart form encoding as per the GA4GH WES Spec.

**Review by ericenns — CHANGES_REQUESTED** [review]
Just a minor comment

File: docs-site/docs/configuration/ga4gh_wes.md (hunk 1 above)
Comment (JeffreyThiessen) [reply docs-site/docs/configuration/ga4gh_wes.md]: Fixed in https://github.com/phac-nml/irida-next/pull/628/commits/ccc745a870e8897af47c5cc4221ed50e20a12cd5

File: docs-site/docs/configuration/ga4gh_wes.md [hunk 2]
```diff
@@ -4,34 +4,72 @@ id: ga4gh_wes
...
 ```yml
 ga4gh_wes:
+  server_url_endpoint: 'https://<some server url>/wes/1.01/'
   oauth_token: <some oauth token>
   headers: { '<some header key>': '<some header value>' }
-  server_url_endpoint: 'https://<some server url>/wes/1.01/'
 ```
 
-#### oauth_token
+Only `server_url_endpoint` is required by default, but depending on your GA4GH WES server configuration the others might also be needed.
```
Comment (deepsidhu85) [inline docs-site/docs/configuration/ga4gh_wes.md:26]: Can we change this to:

Only `server_url_endpoint` is required by default, but depending on your GA4GH WES server configuration the other fields may also be required.

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
Looks great just one comment below

File: docs-site/docs/configuration/ga4gh_wes.md (hunk 2 above)
Comment (JeffreyThiessen) [reply docs-site/docs/configuration/ga4gh_wes.md]: Fixed in https://github.com/phac-nml/irida-next/pull/628/commits/5692b2afe9fb6da61e753dfe1d2d26dec66c4221

**Review by deepsidhu85 — APPROVED** [review]
Looks good to me!

**Pull request merged**